src/content/popup/KanjiReferencesTable.fixture.tsx

```diff
@@ -0,0 +1,176 @@
+import { KanjiReferencesTable } from './KanjiReferencesTable';
+
+// import '../../../css/popup.css';
+
+const TEST_ENTRY = {
+  'c': '日',
```

Comment:
I think if we ran this through prettier it would simplify this to just, `c: '日'` for example. We could also wrap a bunch of the lines up to make it easier to read.